.github/workflows/build_wheels_linux.yml

```diff
@@ -35,10 +35,17 @@ on:
         description: "Name of the actual python package that is imported"
         default: ""
         type: string
-      build-type:
-        description: "Type of build: development, nightly, or release"
+      trigger-event:
```

Comment:
Is "push" the only common trigger-event that would trigger this or we expect others to be trigger too?

Reply:
Good question. Both nightly and release builds occur when something is pushed to a particular branch, so I think it is suitable here. For all other trigger types (PR's, workflow-dispatch), I think it should be fine to skip the upload.